packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -454,11 +454,15 @@ export class GumbyController {
         await this.prepareProjectForSubmission(message)
     }
 
-    private transformationFinished(data: { message: string | undefined; tabID: string }) {
+    private transformationFinished(data: {
+        message: string | undefined
+        tabID: string
+        includeStartNewTransformationButton: string
```

Comment:
Should be a `boolean`

Reply:
I had originally had it as a boolean but then kept getting errors where the data buffer passed into fire was of the wrong type and received a boolean (false). Unless there was a different way to get around this ...